jobs/env/ci-kubernetes-e2e-gci-gce-netd.env

```diff
@@ -0,0 +1,123 @@
+### job-env
+KUBE_CUSTOM_NETD_YAML="""
+  kind: ClusterRole
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: netd
+    namespace: kube-system
+    labels:
+      kubernetes.io/cluster-service: "true"
+      addonmanager.kubernetes.io/mode: EnsureExists
+  rules:
+  - apiGroups: [""]
+    resources: ["nodes"]
+    verbs: ["get"]
+  ---
+  kind: ServiceAccount
+  apiVersion: v1
+  metadata:
+    name: netd
+    namespace: kube-system
+    labels:
+      kubernetes.io/cluster-service: "true"
+      addonmanager.kubernetes.io/mode: EnsureExists
+  ---
+  kind: ClusterRoleBinding
+  apiVersion: rbac.authorization.k8s.io/v1
+  metadata:
+    name: netd
+    labels:
+      kubernetes.io/cluster-service: "true"
+      addonmanager.kubernetes.io/mode: EnsureExists
+  roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: netd
+  subjects:
+  - kind: ServiceAccount
+    name: netd
+    namespace: kube-system
+  ---
+  kind: ConfigMap
+  apiVersion: v1
+  metadata:
+    name: netd-cni-config
+    namespace: kube-system
+    labels:
+      addonmanager.kubernetes.io/mode: EnsureExists
+  data:
+    cni_network_config: |-
+      {
+        "name": "k8s-pod-network",
+        "cniVersion": "0.3.1",
+        "plugins": [
+          {
+            "type": "ptp",
+            "mtu": 1460,
+            "ipam": {
+                "type": "host-local",
+                "subnet": "%s",
```

Comment:
for subnet, please change it to ranges

https://github.com/containernetworking/plugins/tree/master/plugins/ipam/host-local